### ISSUE TITLE
[Responsive Navigation] Responsive Toggle does not work 

### ISSUE THREAD

**Issue by xiaoshuw** [zurb/foundation-sites#7150]
**How can we reproduce this bug?**

1. Follow the example to build responsive toggle

**What did you expect to happen?**

Expect the toggle between title-bar and menu. 

**What happened instead?**

There are a few bizarre cases that I found on various browser.
(1) On Chrome browser, if include the foundation.js from bower_components/bower_components/foundation-sites/dist/, it will work as expected. However, if included "foundation.min.js", it will not work.
(2) Even using foundation.js, responsive toggle does not work on either Safari or Firefox. The title-bar shows all the time and click on menu bar does nothing.
(3) I tried to use the foundation.js from your websites, that seemed have worked. 

**Test case**

Give us a link to a [CodePen](http://codepen.io) or [JSFiddle](http://jsfiddle.net) that recreates the issue.

**Comment by rafibomb** [comment]
Can you try to run bower update and see if it still happens. We made some changes to the minified version on Friday. Bower update will pull the latest patch (6.0.2)

**Comment by xiaoshuw** [comment]
I think my browser is updated. I have tried to minimize the foundation.js with some library. Neither YUI or closoure-compiler works. The one compiled with uglify-js works on chrome. 

I tried to download the zip file (essential and full) instead of using the one installed by foundation cli. It has similar result. The only foundation.js that worked (on all browsers) was the one downloaded from the link of your document site, i.e., http://foundation.zurb.com/sites/docs/assets/js/foundation.js
